### ISSUE TITLE
Something is being cached improperly? 

### ISSUE THREAD

**Issue by edjiang** [stormpath/express-stormpath#330]
In `express-stormpath 2.4.0`: 

1. Create a new user w/ email verification enabled.
2. Verify the user via the email link, then POST against `/oauth/token` to get an access token
3. Try to get `/me`. Response is `401 Unauthorized`
4. Restart the server. Use the same access token. Response is `200 OK`

**Comment by edjiang** [comment]
Try to reproduce in framework spec upgrade branch

**Comment by edjiang** [comment]
@robertjd just tried to reproduce, still happens

**Comment by robertjd** [comment]
Hey @edjiang I'm not able to reproduce this.  Here is what I'm doing:

From my browser:

1. registering for an account
2. clicking on the email I get, to verify the account

Then from my REST tool:

3. Post to `/oauth/token` with my new account's username and password, I get back an access token
4. I use that token with `Authorization: Bearer <token>` when requesting `/me`, and I get the expected account response back.

Am I doing something differently?

**Comment by edjiang** [comment]
I registered via POST, not sure if that should make a difference. I'll try reproducing again

**Comment by robertjd** [comment]
> I registered via POST, not sure if that should make a difference. I'll try reproducing again

What client/client library are you using? Not a browser?

**Comment by edjiang** [comment]
Huh, I was able to do that from the browser as well. Let me follow up with you on chat.

**Comment by edjiang** [comment]
Here's me trying to reproduce it from scratch:

Install the `express-stormpath-sample-project`. Change `package.json`'s dependency to:

`"express-stormpath": "stormpath/express-stormpath#framework-spec-upgrade",`

Then `npm install`. 

Repeat steps from initial post for the issue. It breaks for me.

**Comment by robertjd** [comment]
Can you please clarify what type(s) of client you are using for each step?

**Comment by edjiang** [comment]
It happened when I was both using the website, and posting json. But here's the json requests I was sending:

```
POST http://localhost:3000/register
Content-Type:application/json
Accept:application/json
X-Stormpath-Agent:stormpath-sdk-swift/1.1
```

```
POST http://localhost:3000/oauth/token
Accept:application/json
X-Stormpath-Agent:stormpath-sdk-swift/1.1
```

```
GET http://localhost:3000/me
Accept:application/json
X-Stormpath-Agent:stormpath-sdk-swift/1.1
Authorization:Bearer XYZ
```

**Comment by edjiang** [comment]
@robertjd and I did a call, and we figured out why it was happening for me. I was verifying against `https://api.stormpath.com/emailVerificationTokens?sptoken=`.

The `/oauth/token` endpoint forwards the request to Stormpath, so the account cache doesn't get hit and `express-stormpath` returns a valid token. 

However, since the account is cached as UNVERIFIED, when trying to hit the endpoints, the application will return `401 Unauthorized`.

Changing the link base URL for email verification to express fixes this. 

I'm gonna close this for now, but @omgitstom if people using our framework integrations should not be verifying against `api.stormpath.com`, we should probably change the default so it gets set properly.
